### ISSUE TITLE
Use with Vue

### ISSUE THREAD

**Issue by rkrejcii** [Hagsten/Talkify#54]
Hello,

I would like to use Talkify with Vue.

I installed Talkify using

```
$ npm install talkify-tts
```

but then, when I use in my Vue single file component (https://vuejs.org/v2/guide/single-file-components.html) this import command:

```
import talkify from 'talkify-tts';
```

`talkify` is only an empty object, so things like:

```
 var player = new talkify.TtsPlayer().enableTextHighlighting();
```

are obviously not working. What I'm doing wrong please?

Thank you for any help.

**Comment by Hagsten** [comment]
Hi,

I am not familiar with Vue, but from our old React repo (https://github.com/Hagsten/TalkifyReactJs/blob/master/src/lib/talkify.js) we use

import 'talkify-tts';

Talkify attaches itself to the global scope. Perhaps this helps.

**Comment by rkrejcii** [comment]
Thank you very much for your quick reply.

`window.talkify` works, but I'm facing other problems now.

First, I tried (inside my Vue component) this code:

```
      const talkify = window.talkify;
      const player = new window.talkify.Html5Player();

      player.playText('Hello world');
```

It failed with following error:

```
TypeError: Cannot read property 'Promise' of undefined
```

The problem was in this one snippet (from `talkify.js`):

```
      function getVoice() {
          var p = new promise.promise.Promise();
```

After following modification (one `promise` removal), so:

```
      function getVoice() {
          var p = new promise.Promise();
```

the code works and "Hello world" was spoken using the voice set in my system.

Is your code wrong or the problem is related to my environment?

Second problem:

This code:

```
      const talkify = window.talkify;
      const player = new window.talkify.Html5Player().enableTextHighlighting();

      const playlist = new talkify.playlist()
        .begin()
        .usingPlayer(player)
        .withTextInteraction()
        .withElements(document.querySelectorAll('p')) //<--Any element you'd like. Leave blank to let Talkify make a good guess
        .build();
    
        playlist.play(); 
```
fails with 

```
TypeError: Cannot read property 'indexOf' of undefined
```

The problem is related to following snippet (from `talkify.js`):

```
      function chunckText(text) {
        var language = me.settings.lockedLanguage || me.settings.referenceLanguage.Culture;
        var chunckSize = language.indexOf('zh-') > -1 ? 70 : language.indexOf('ko-') > -1 ? 130 : 200;
```

When i use `console.log(me.settings)`, I'm getting following info:

![image](https://user-images.githubusercontent.com/3452307/107540927-e08d8600-6bc6-11eb-8e1a-d7657a3b9523.png)

Could you give me please any hint what's wrong here?

Thanks a lot.

**Comment by Hagsten** [comment]
Hi,

I'm on my phone now so my answer will be brief and might not be accurate. 

I see that you are using the local HTML player. In that case you should specify which voice or culture to use. You can use either forceVoice or forceLanguage on your HTML player instance.

Regarding the promise it seems odd to use the repetitive version. I can confess that the HTML player has been a second class citizen. 

Perhaps it is time to get rid of that dependency and use the native promise instead. 

Anyway, specifying a voice will probably solve the problem.

**Comment by Hagsten** [comment]
The promise issue should have been resolved in v3.9.0. Please reopen if this is still an issue. Closing for now.
